docs/common_hal.md

```diff
@@ -0,0 +1,70 @@
+# Adding `nativeio` support to other ports
```

Comment:
Hi @tannewt. You are very welcome! No worries on the title. Just so you know for the future, I'm pretty mellow about suggestions that I make. Basically, use those that make sense and pass on the others. :smile: Thanks!